plugins/admin/modules/helper.go

```diff
@@ -39,7 +40,8 @@ func RemoveBlankFromArray(s []string) []string {
 }
 
 func Uuid() string {
-	return uuid.NewV4().String()
```

Comment:
See the latest release version of package go.uuid, https://github.com/satori/go.uuid/blob/v1.2.0/generator.go#L64.

Reply:
It work fine when I update the package go.uuid, thanks.

Reply:
#77